util.go

```diff
@@ -0,0 +1,58 @@
+package lo
+
+// Range creates an array of numbers (positive and/or negative) with given length.
+func Range(elementNum int) []int {
+	if elementNum >= 0 {
+		result := make([]int, elementNum)
+		for i := 0; i < elementNum; i++ {
+			result[i] = i
+		}
+		return result
+	}
+	result := make([]int, -elementNum)
+	for i := 0; i < -elementNum; i++ {
+		result[i] = -i
+	}
+	return result
+}
+
+// RangeFrom creates an array of numbers from start with specified length.
+func RangeFrom(start, elementNum int) []int {
+	if elementNum >= 0 {
+		result := make([]int, elementNum)
+		for i := 0; i < elementNum; i++ {
+			result[i] = i + start
+		}
+		return result
+	}
+	result := make([]int, -elementNum)
+	for i := 0; i < -elementNum; i++ {
+		result[i] = start - i
+	}
+	return result
+}
+
+// RangeOpen creates an array of numbers (positive and/or negative) progressing from start up to, but not including end.
+// step is default 1 when set to zero or can not reach end.
+func RangeOpen(start, end, step int) []int {
```

Comment:
`func RangeOpen[T constraints.Integer | constraints.Float](start T, end T, step T) []T {`

Reply:
Hmm...I don't really think `RangeOpen` is a good name for this. Maybe something like `RangeSteps` or `RangeWithSteps`? I feel like `RangeOpen` isn't really descriptive enough. 

WDYT?

Reply:
May be RangeWithSteps is appropriate for this 😂

Reply:
RangeFromWithSteps ? 😆 

I like RangeWithSteps too